custom_components/omnik_inverter/diagnostics.py

```diff
@@ -10,9 +10,9 @@
 from homeassistant.core import HomeAssistant
 
 from . import OmnikInverterDataUpdateCoordinator
-from .const import DOMAIN, SERVICE_DEVICE, SERVICE_INVERTER
+from .const import CONF_SERIAL, DOMAIN, SERVICE_DEVICE, SERVICE_INVERTER
 
-TO_REDACT = {CONF_HOST, CONF_IP_ADDRESS}
+TO_REDACT = {CONF_HOST, CONF_IP_ADDRESS, CONF_SERIAL}
```

Comment:
I see no reason to add this.

Reply:
Why would we redact the IP address but not a device-specific serial number?

Reply:
I was a bit confused by the way Github put the const.py above, so it seems like you're redacting a label and not the value in that label 😅

Reply:
It seems like `async_redact_data` acts on a mapping of key, replacing every key in this `TO_REDACT` list with the value of `"**REDACTED**"` :)